app/modules/users/models.py

```diff
@@ -470,7 +472,7 @@ def is_email_confirmed(self):
     def picture(self):
         from app.modules.assets.models import Asset
 
-        asset = Asset.query.filter_by(id=self.profile_asset_guid).first()
+        asset = Asset.query.filter_by(id=self.profile_fileupload_guid).first()
```

Comment:
is this Asset (vs FileUpload) just leftover from pre-fileupload?  maybe this `picture()` method is actually cruft that can be retired now.

Reply:
Oh...!  I didn't notice this.  Let me remove this method.

Reply:
yeah its weird -- i didnt notice this before either. haha  snuck by us both.